db/migrate/20221004122343_add_dark_mode_logo_to_categories.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddDarkModeLogoToCategories < ActiveRecord::Migration[7.0]
+  def change
+    add_column :categories, :logo_dark_url, :string
```

Comment:
Is this `logo_dark_url` column being used?

Reply:
Maybe you forgot to push that commit? That line is still there. It also needs to be removed from the schema information in the `category.rb` model file.

Reply:
I was waiting for your response on the other topic to do so. My comment was a little misleading, though. I'll try to be more clear in the future.
All changes are pushed now.